coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlIdentifierTransformer.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.common.transformers;
+
+import org.apache.calcite.sql.SqlIdentifier;
+
+
+/**
+ * This is the abstract class which defines the transformation of the function inside SqlIdentifier
+ */
+public abstract class SqlIdentifierTransformer {
```

Comment:
I think this transformer was introduced for the corner case `f(x).y.z` ([detailed comment](https://github.com/linkedin/coral/pull/351/files#diff-c5c8865e887e815b03a63a0de8fe443d3e7eb0b0a913e9920255e9dc55888995R353))?
I would suggest to override this behavior in `CoralRelToSqlNodeConverter` like how #351 does, so that we don't need to create this special class for such a corner case and specific UDF. cc: @wmoustafa

Reply:
Good point.  The function of `toSql` is overridden in the class of `RelToTrinoConverter` similar to the code in #351. And the classes to convert the function in SqlIdentifier are removed.